linode/manager
# Add: Splash Screen

## Description

Adds a Splash Screen that shows before relevant data has loaded, so the user isn't looking at a blank screen indefinitely.

## Type of Change
- Non breaking change ('update', 'change')

## Applicable E2E Tests

N/A

## How it Works

1. User navigates to App and `main.js` is requested
2. When the user is authed, we request all things and start the events polling
   * App.tsx is code-splitted but at this point has loaded and is hidden with CSS
3. `DataLoadedListener` will observe what data has loaded and check the `location.pathname` to see if the relevant data has loaded.
4. App screen shows

## To Test

Open dev tools and try loading the app with all kinds of different network throttling. Try disabling caching to see what results you get.
